src/check_synced.rs

```diff
@@ -0,0 +1,68 @@
+//! Validates that remote data matches data in team repo.
+
+use crate::data::Data;
+use crate::github::{self, GitHubApi};
+use crate::schema;
+use log::error;
+use rayon::prelude::*;
+use std::collections::HashMap;
+
+pub(crate) fn check(data: &Data) -> Result<(), failure::Error> {
+    const BOT_TEAMS: &[&str] = &["bors", "bots", "rfcbot", "highfive"];
+    let github = GitHubApi::new();
+    let mut remote_teams = github
+        .teams()?
+        .into_par_iter()
```

Comment:
IIRC, GitHub recommends against concurrent requests to their API, but I suppose it seems OK for our purposes. Is this a major win in practice? I know the API can be rather slow, so it very well may be, but given the added dependency on rayon I'd like to make sure it's worth it (e.g., shaving dozens of seconds off).